Forms automated test - Master | LPS-155762

vsigor opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

SelenaAungst committed:
LPS-155993 Modified language key.

jkappler committed:
COMMERCE-9502 Follow same approach as other places and wait for PortletRepository to be available and avoid an UndeployedExternalRepositoryException

Commit:
COMMERCE-8078 Add AssertSVGTypeDiagramStoreFront

Commit:
COMMERCE-8078 Update changePinInfo macro

Commit:
COMMERCE-8078 Update test with changePinInfo

nikki-pru committed:
COMMERCE-8078 Reduce variability for more stable tests

victorg1991 committed:
LPS-156391 Add custom css component

victorg1991 committed:
LPS-156391 Language keys

victorg1991 committed:
LPS-156391 buildLang

victorg1991 committed:
LPS-156391 Add customCSS property in the StyledLayoutStructureItem

victorg1991 committed:
LPS-156391 Save customCSS when the user has introduced something

miltonmc committed:
LPS-156582 Enforce naming consistency

michaelhashimoto committed:
LRCI-2991 Avoid null pointer when using System.out in CI

michaelhashimoto committed:
LRCI-2991 Add logic for JenkinsResultsParserUtil.read() to gzip files

michaelhashimoto committed:
LRCI-2991 Fix off by one error when getting Build Result JSON URLs

michaelhashimoto committed:
LRCI-2991 Fix the order of the JSONObjects returned from JenkinsResultsParserUtil

michaelhashimoto committed:
LRCI-2991 Make 'Build.getStopWatchRecordsGroup()' a public method

michaelhashimoto committed:
LRCI-2991 Add JobReport class to gather BuildReport objects

jkappler committed:
Revert "LPS-148819 Assert partial text"

This reverts commit 47181bf57c3c2a00c8f905406ac3328e356af4c3.

jkappler committed:
Revert "LPS-148819 Fix another path"

jkappler committed:
Revert "LPS-148819 Update assertions"

This reverts commit 38c4a78be6d9307a949d3052237f61ec265ed0bc.

jkappler committed:
Revert "LPS-148819 Include alternative paths for different views"

This reverts commit 515769444cae1cc7304c8df18a8978c1e61868b2.

jkappler committed:
Revert "LPS-148819 Update blogs content test locator path"

This reverts commit dcdca8924b14621c19fa96f7ad676df9f0fe2c88.

jkappler committed:
Revert "LPS-148819 Change the defaults behavior of the editor to CKEDITOR.ENTER_DIV (2)"

This reverts commit 165b24d72172927b37d6048525629e0e601ddc56.

cgoncas committed:
LPS-152541 Add new API method updateSegmentsConfiguration to SegmentsConfigurationProvider.

cgoncas committed:
LPS-152541 Add implementation to SegmentsConfigurationProviderImpl.

cgoncas committed:
LPS-152541 Test it

cgoncas committed:
LPS-152541 New method isSegmentsCompanyConfigurationDefined in SegmentsConfigurationProvider

cgoncas committed:
LPS-152541 Implement it.

cgoncas committed:
LPS-152541 Test it.

victorg1991 committed:
LPS-156391 Extract

AliciaGarciaGarcia committed:
LPS-156379 use escapeAttribute to escape file description on SVG files

victorg1991 committed:
LPS-156391 Save customCSS values per viewport

victorg1991 committed:
LPS-156391 Show advanced panel in other viewports too

victorg1991 committed:
LPS-156391 Don't show Hide from search option in smaller viewports

victorg1991 committed:
LPS-156391 Add support for saving customCSS in different viewports

victorg1991 committed:
LPS-156391 Since hide fragment is not being show on desktop, we don't need margin here

victorg1991 committed:
LPS-156391 Fix textarea for firefox

User agent stylesheet doesn't have proper styles

victorg1991 committed:
LPS-156391 Baseline

fortunatomaldonado committed:
LPS-155993 Ran buildLang

michaelhashimoto committed:
LRCI-2991 Add BuildReport class hierarchy to gather build data without a Build object

michaelhashimoto committed:
LRCI-2991 Add TestReport class hierarchy to gather test data without a TestResult object

michaelhashimoto committed:
LRCI-2991 Record the 'build-report.json.gz' file instead of 'build-result.json' in Testray

michaelhashimoto committed:
LRCI-2991 SF

welitonpe committed:
LPS-156220 Created a feature to can sort a table

michaelhashimoto committed:
LRCI-2291 Generate the CI System History report from BuildReport objects

michaelhashimoto committed:
LRCI-2991 Use the actual start timestamp if available for StopWatchRecord objects

michaelhashimoto committed:
LRCI-2991 Use the BuildReport objects to get the test results json objects

michaelhashimoto committed:
LRCI-2991 Always delete the local build database file when running locally

dsitu committed:
COMMERCE-9476 Remove mimium instance name space from knowledge base portlet

kyle-miho committed:
LRQA-71312 Fix PQL logic and increase coverage since testing changes to portal-impl and portal-kernel should be more strict for the relevant suite

kyle-miho committed:
LRQA-71312 Fix PQL logic for running relevant tests when making changes to Poshi tests

michaelhashimoto committed:
LRCI-2991 SF

michaelhashimoto committed:
LRCI-2991 Shift over recording flaky tests to BuildReports if available

miltonmc committed:
LPS-156581 Disable placeholder selection

victorg1991 committed:
LPS-156391 Adapt tests

kyle-miho committed:
LRQA-75217 Use job name since portal-release does not cover integration tests, use the base job over the suite name

kyle-miho committed:
LRQA-75217 Fix wrong wildcards and typo

marcelabc committed:
LPS-156587 When adding an entry, verify if it is null/empty

Hanlf committed:
LPS-156610 Google Drive shortcut is hidden when not configured

Hanlf committed:
LPS-156610 Update description and remove property

ling-alan-huang committed:
LPS-150378 SF, inline

ling-alan-huang committed:
LPS-150378 SF, inline

ling-alan-huang committed:
LPS-150378 SF, inline

Hijky committed:
LPS-156477 Add a test to can get ManyToMany relationship details as nested fields_4

Commit:
LPS-156101 Add case to configure Date fragment

izaera committed:
LPS-156515 Make CETConfigurationFactory refresh CETManager when configuration is updated

jkappler committed:
LPS-156391 Simplify

victorg1991 committed:
LPS-155220 Fix disableonchecked behaviour

This also makes surt that the default behaviour is preserved, so when the attribute is not present the default value is true

brianchandotcom committed:
Revert "LPS-155410 make sure to undeploy any previous CET registrations"

This reverts commit c2b7d4c814a4b57d93bd6564dbd57b8a264534be.

brianchandotcom committed:
LRCI-2991 prep next

ealonso committed:
LPS-156641 Keep the selected value on input field

Commit:
LCD-14360 ensure the configuration plugin is available as early as possible

brianchandotcom committed:
LCD-14360 Wordsmith

brianchandotcom committed:
artifact:ignore object-web 1.0.81 prep next

Commit:
LPS-155762 Add ViewMultipleSelectionFieldDetails test
